spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -1348,57 +1372,92 @@
+                it 'deactivates the associated profile' do
+                  expect(pending_enrollment.profile.in_person_verification_pending_at).not_to be_nil
 
-                job.perform(Time.zone.now)
+                  job.perform(Time.zone.now)
+                  pending_enrollment.reload
+                  expect(pending_enrollment.profile.in_person_verification_pending_at).to be_nil
+                  expect(pending_enrollment.profile.active).to be false
+                  expect(pending_enrollment.profile.deactivation_reason).
+                    to eq('verification_cancelled')
+                end
 
-                profile = pending_enrollment.reload.profile
-                expect(profile).not_to be_active
-                expect(profile.fraud_review_pending_at).to be_nil
-                expect(profile.fraud_rejection_at).not_to be_nil
-                expect(job_analytics).to have_logged_event(
-                  :idv_ipp_deactivated_for_never_visiting_post_office,
-                )
+                it 'deactivates and sets fraud related fields of an expired enrollment' do
+                  stub_request_expired_id_ipp_proofing_results
+
+                  job.perform(Time.zone.now)
+
+                  profile = pending_enrollment.reload.profile
+                  expect(profile).not_to be_active
+                  expect(profile.fraud_review_pending_at).to be_nil
+                  expect(profile.fraud_rejection_at).not_to be_nil
+                  expect(job_analytics).to have_logged_event(
+                    :idv_ipp_deactivated_for_never_visiting_post_office,
+                  )
+                end
               end
             end
           end
         end
 
         describe 'Proofed with secondary id' do
-          let(:pending_enrollment) do
-            create(
-              :in_person_enrollment, :pending
-            )
+          let!(:pending_enrollments) do
+            ['BALTIMORE', 'FRIENDSHIP', 'WASHINGTON', 'ARLINGTON', 'DEANWOOD'].map do |name|
+              create(
+                :in_person_enrollment,
+                :pending,
+                :with_notification_phone_configuration,
+                issuer: 'http://localhost:3000',
+                selected_location_details: { name: name },
+                sponsor_id: usps_ipp_sponsor_id,
+              )
+            end
+          end
+          let(:pending_enrollment) { pending_enrollments.first }
+
+          before do
+            enrollment_records = InPersonEnrollment.where(id: pending_enrollments.map(&:id))
+            # Below sets in_person_verification_pending_at
+            # on the profile associated with each pending enrollment
+            enrollment_records.each do |enrollment|
+              profile = enrollment.profile
+              profile.update(in_person_verification_pending_at: enrollment.created_at)
+            end
+            pending_enrollment.reload
+            # binding.pry
```

Comment:
Please delete the breakpoint, thanks.

Reply:
resolved with commit 6cf768e8990f2722d382f5cb1390be89ff1ae0b4